pandas/tests/io/test_pytables.py

```diff
@@ -736,6 +736,20 @@ def test_put_compression_blosc(self):
             store.put('c', df, format='table', complib='blosc')
             tm.assert_frame_equal(store['c'], df)
 
+    def test_complibs(self):
+        df = tm.makeDataFrame()
+
```

Comment:
look in the meta data of the file
lots of _ attributes that define things like this
pytables docs might be helpful